### DISCUSSION TITLE
Pylab的module從哪裡下載呢?

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#31] [Q&A] [answered]
有一個問題想問！
在微積分課程有python作業，要import pylab來顯示函式圖片
但是我在visual studio code內測試時，他說我沒有pylab的module
所以想問問各位，那個module是要從哪裡下載呢?   • o •

**Comment by Yunhans** [comment, marked as answer]
我是直接在終端機輸入
pip3 install matplotlib

**Reply by bonny123865** [reply]
感謝！我試成功了！

**Comment by chia-kai-chang** [comment]
安裝`matplotlib`就會順便把`pylib`安裝進去
windows用戶
`pip install matplotlib`

**Reply by bonny123865** [reply]
好的好的，謝謝老師！！已經成功安裝了！
